luaui/Widgets/map_startpolygon_gl4.lua

```diff
@@ -0,0 +1,227 @@
+function widget:GetInfo()
```

Comment:
This whole file seems to be a copy-paste of the change to `map_startbox.lua` above.

Reply:
Its the other way around.